documentation/tutorials/get-started-with-phoenix.md

```diff
@@ -0,0 +1,467 @@
+# Get Started
+
+<!--- ash-hq-hide-start --> <!--- -->
+
+This documentation is best viewed at [ash-hq.org](https://ash-hq.org)
+
+<!--- ash-hq-hide-stop --> <!--- -->
+
+## Goals
+
+In this guide we will:
+
+1. Create a new Phoenix application and add Ash, AshPhoenix and AshPostgres as dependencies
+2. Create a simple set of resources and show they can be used
+3. Illustrate some core concepts of Ash
+4. Point you to good next resources so you can explore Ash further
+
+## Things you may want to read first
+
+- [Install Elixir](https://elixir-lang.org/install.html)
+- [Phoenix - Up and Running Guide](https://hexdocs.pm/phoenix/up_and_running.html)
+- [Philosophy Guide](/documentation/tutorials/philosophy.md)
+
+## Requirements
+
+If you want to follow along yourself, you will need the following things:
+
+1. Elixir (1.12 or later) and Erlang (22 or later) installed
+2. PostgreSQL installed
+3. A text editor to make the changes that we make
+4. A terminal to run the examples
+
+## Steps
+
+For this tutorial, we'll use examples based around creating a help desk.
+
+We will make the following resources:
+
+- `PhoenixHelpdesk.Support.Ticket`
+- `PhoenixHelpdesk.Support.Representative`
+
+The actions we will be able to take on these resources include:
+
+- Opening a new Ticket
+- Closing a Ticket
+- Assigning a Ticket to a representative
+
+### Create a New Phoenix Project
+
+_This section is based the [Phoenix installation docs](https://hexdocs.pm/phoenix/installation.html). For more details go there._
+
+We first need to create a fresh phoenix project using the phoenix project generator.
+
+```bash
+mix archive.install hex phx_new
+```
+
+And then use the project generator.
+
+```bash
+mix phx.new phoenix_helpdesk
+```
+
+This will then generate a project boilerplate and ask if you want to fetch and install dependencies. **Answer 'Yes' to 'Fetch and install dependencies? [Yn]'**.
+
+Your terminal should now look very similar to this:
+
+```bash
+* creating phoenix_helpdesk/priv/static/images/phoenix.png
+* creating phoenix_helpdesk/priv/static/favicon.ico
+
+Fetch and install dependencies? [Yn] y
+* running mix deps.get
+* running mix deps.compile
+
+We are almost there! The following steps are missing:
+
+    $ cd phoenix_helpdesk
+
+Then configure your database in config/dev.exs and run:
+
+    $ mix ecto.create
+
+Start your Phoenix app with:
+
+    $ mix phx.server
+
+You can also run your app inside IEx (Interactive Elixir) as:
+
+    $ iex -S mix phx.server
+```
+
+Please `cd` into the project directory **BUT DO NOT RUN `mix ecto.create`.** We first need to integrate Ash and AshPostgres into our project.
```

Comment:
Maybe this should go above the `$mix ecto.create` line above? I would probably have run that first.

Reply:
We also have `mix ash_postgres.create` (we have tasks for all of the ecto tasks that should be used instead)

Reply:
which I see is clear down below

Reply:
addressed: fb62b925997118b19a79678abf492252ec821c4d